src/Http/Routing/src/EndpointRoutingMiddleware.cs

```diff
@@ -33,13 +35,15 @@ internal sealed partial class EndpointRoutingMiddleware
         EndpointDataSource rootCompositeEndpointDataSource,
         DiagnosticListener diagnosticListener,
         IOptions<RouteOptions> routeOptions,
+        RoutingMetrics metrics,
```

Comment:
Should the `RoutingMetrics` class be added to `IServiceCollection` in `AddRouting`?